RELEASE/scripts/sl_ascend.ash

```diff
@@ -3904,7 +3904,8 @@ boolean L11_palindome()
 			{
 				use(1, $item[&quot;2 Love Me\, Vol. 2&quot;]);
 				doHottub();
-				bat_reallyPickSkills(20);
+				if(my_class() == $class[Vampyre])
+					bat_reallyPickSkills(20);
 			}
```

Comment:
Oooooh, that's how that was happening? That makes sense, actually. Can you move that check in to bat_reallyPickSkills though instead of having it there?

Reply:
Actually I'll do it myself.